### PR TITLE
Add changes to citron branch

### PR DESCRIPTION
**Pull request by citron-kathleen**
Please review my code, Thank you.

### CODE REVIEW

File: bnpackage/citron.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+def kathleen_citron ():
+
+    choice = 0
+
+    while choice != 4:
+
+        print(f"Hello! My name is Kathleen C. Citron")
```
Comment (ynion-mabeamae) [inline bnpackage/citron.py]: Please remove the "f".

File: bnpackage/citron.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        print("1. Basic Info")
+        print("2. Goals")
+        print("3. Comments")
+        print("4. Exit")
+        choice = int(input("Enter your choice: "))
+
+        match choice:
+
+            case 1:
+                print(f"i like vegetables.")
```
Comment (delima-justine) [inline bnpackage/citron.py]: Remove f-string and make 'i' uppercase 'I'.

File: bnpackage/citron.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+                print(f"i'm 19 years old.")
+                pint(f"i love playing online games.")
```
Comment (relente-patriciajoy) [inline bnpackage/citron.py]: Please run your code first and check the error. Remove f-string too.

**Review by relente-patriciajoy — CHANGES_REQUESTED** [review]
Please check your code before I approve it.

File: bnpackage/citron.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+            case 2:
+                print(f"my goal is to be a successful person.")
+                print(f"improving my codingg skills.")
+            case 3:
+                print(f"opinionsss:")
```
Comment (delima-justine) [inline bnpackage/citron.py]: Remove extra 's' in opinions and make 'o' uppercase 'O'

File: bnpackage/citron.py (hunk 3 above)
Comment (ynion-mabeamae) [inline bnpackage/citron.py]: Please remove the 'f' string and capitalized the letter I.

File: bnpackage/citron.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        match choice:
+
+            case 1:
+                print(f"i like vegetables.")
+                print(f"i'm 19 years old.")
+                pint(f"i love playing online games.")
+
+            case 2:
+                print(f"my goal is to be a successful person.")
+                print(f"improving my codingg skills.")
```
Comment (ynion-mabeamae) [inline bnpackage/citron.py]: Please remove the 'f' and capitalized the word "my" and "improving" into "My" and "I".

File: bnpackage/citron.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        print("2. Goals")
+        print("3. Comments")
+        print("4. Exit")
+        choice = int(input("Enter your choice: "))
+
+        match choice:
+
+            case 1:
+                print(f"i like vegetables.")
+                print(f"i'm 19 years old.")
```
Comment (delima-justine) [inline bnpackage/citron.py]: Make "im" to "I'm"

File: bnpackage/citron.py (hunk 4 above)
Comment (ynion-mabeamae) [inline bnpackage/citron.py]: Check your spelling.

**Review by ynion-mabeamae — CHANGES_REQUESTED** [review]
Please check your code before I approved.

**Review by delima-justine — CHANGES_REQUESTED** [review]
Great job but make the first lowercase letter to uppercase, thank you!

File: bnpackage/citron.py (hunk 3 above)
Comment (quiambao-mapatriciaanne) [inline bnpackage/citron.py]: Please capitalized the word "I".

**Review by quiambao-mapatriciaanne — CHANGES_REQUESTED** [review]
Please check your code before I approve it.

**Review by ynion-mabeamae — APPROVED** [review]
Good job!

**Review by quiambao-mapatriciaanne — APPROVED** [review]
Good job!

**Review by delima-justine — APPROVED** [review]
Great!

**Review by relente-patriciajoy — APPROVED** [review]
Amazing

**Pull request merged**